multiaccounts/settings/database.go

```diff
@@ -242,6 +252,10 @@ func (db *Database) SaveSettingField(sf SettingField, value interface{}) error {
 	return db.parseSaveAndSyncSetting(sf, value)
 }
 
+func (db *Database) EmptyMnemonic() error {
+	return db.saveSetting(Mnemonic, nil)
+}
+
```

Comment:
Perhaps change to `DeleteMnemonic` just so that the terminology is aligned with CRUD conventions.

Reply:
fixed